demo/src/layouts/index.js

```diff
@@ -1,6 +1,21 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import MainLayout from './MainLayout';
 
-const Layout = (props) => <MainLayout {...props} />;
+const Layout = (props) => {
+  const { children, pageContext } = props;
+
+  switch (pageContext.layout) {
+    case 'basic':
+      return children;
+    default:
+      return <MainLayout {...props} />;
+  }
+};
```

Comment:
Question: do we suspect there will be other cases where we would like to render layout differently? Curious why you reached for a `switch` here vs an early return.

Reply:
This is the demo site. To be honest, I didn't put much thought into it. I just copied it from here:

https://github.com/newrelic/docs-website/blob/3e9e090016d4084546c49365d0763591c0aee468/src/layouts/index.js#L8-L13

Reply:
Ultimately this will be implemented by the sites themselves. This layout is only for the demo site, which has its own layout (which currently mimics the current docs/dev sites)